projects/plugins/jetpack/extensions/blocks/ai-assistant/hooks/use-suggestions-from-ai/index.ts

```diff
@@ -83,7 +78,24 @@ export default function useSuggestionsFromAI( {
 	// Store the event source in a ref, so we can handle it if needed.
 	const source = useRef< SuggestionsEventSource | undefined >( undefined );
 
-	const readyToRequest = postId && prompt?.length;
+	/**
+	 * onSuggestion function handler.
+	 *
+	 * @param {string} suggestion - The suggestion.
+	 * @returns {void}
+	 */
+	const handleSuggestion = useCallback(
+		( event: CustomEvent ) => onSuggestion( event?.detail ),
+		[ onSuggestion ]
+	);
+
+	/**
+	 * onDone function handler.
+	 *
+	 * @param {string} content - The content.
+	 * @returns {void}
+	 */
+	const handleDone = useCallback( ( event: CustomEvent ) => onDone( event?.detail ), [ onDone ] );
```

Comment:
```suggestion
	const handleDone = useCallback( ( event: CustomEvent ) => onDone?.( event?.detail ), [ onDone ] );
```

`onDone` is not required

Reply:
Ah, just saw that it is only added as listener callback if `onDone` exists.